engine/api/auth/ldapclient.go

```diff
@@ -225,6 +225,28 @@ func (c *LDAPClient) Bind(username, password string) error {
 	return nil
 }
 
+// BindDN
+func (c *LDAPClient) BindDN(dn, password string) error {
+  log.Debug("LDAP> Bind DN %s", dn)
+
+  if err := c.conn.Bind(dn, password); err != nil {
+    if shoudRetry(err) {
+      err = c.openLDAP(c.conf)
```

Comment:
One line err check: if err := c.openldap....

Reply:
I think it isn't that necessary to check the same error openldap in the ldapbind because we check it in the ldapsearch